frontend/javascripts/oxalis/view/action-bar/download_modal_view.tsx

```diff
@@ -0,0 +1,463 @@
+    setIncludeVolumeData(checkedValues.includes("Volume"));
+  };
+
+  const handleKeepWindowOpenChecked = (e: any) => {
+    setKeepWindowOpen(e.target.checked);
+  };
+
+  const workerInfo = (
+    <Row>
+      <Divider
+        style={{
+          margin: "18px 0",
+        }}
+      />
+      <Text
+        style={{
+          margin: "6px 12px",
+        }}
+        type="warning"
+      >
+        {messages["annotation.export_no_worker"]}
+        <a href="mailto:hello@webknossos.com">hello@webknossos.com.</a>
+      </Text>
+    </Row>
+  );
+
+  const checkboxStyle = {
+    height: "30px",
+    lineHeight: "30px",
+  };
+
+  const authToken = useFetch(getAuthToken, "loading...", []);
+  const wkInitSnippet = `import webknossos as wk
+
+with wk.webknossos_context(token="${authToken}"):
+    annotation = wk.Annotation.download(
+        "${annotationId}",
+        annotation_type="${annotationType}",
+        webknossos_url="${window.location.origin}"
+    )
+`;
+
+  const alertTokenIsPrivate = () => {
+    Toast.warning(
+      "The clipboard contains private data. Do not share this information with anyone you do not trust!",
+    );
+  };
+
+  const selection = ["Volume", "Skeleton"];
+
+  return (
+    <Modal
+      title="Download this Annotation"
+      visible={isVisible}
+      width={600}
+      footer={[<Footer tabKey={activeTabKey} onClick={handleOk} key="footer" />]}
+      onCancel={onClose}
+      style={{ overflow: "visible" }}
+    >
+      <Tabs activeKey={activeTabKey} onChange={handleTabChange} type="card">
+        <TabPane tab="Download" key="download">
+          <Row>
+            {maybeShowWarning()}
+            <Text
+              style={{
+                margin: "6px 12px",
+              }}
+            >
+              {messages["annotation.download"]}
+            </Text>
+          </Row>
+          <Divider
+            style={{
+              margin: "18px 0",
+            }}
+          >
+            Options
+          </Divider>
+          <Row>
+            <Col
+              span={9}
+              style={{
+                lineHeight: "20px",
+                padding: "5px 12px",
+              }}
+            >
+              Select the data you would like to download.
+            </Col>
+            <Col span={15}>
+              <CheckboxGroup onChange={handleCheckboxChange} defaultValue={selection}>
+                <Checkbox style={checkboxStyle} value="Volume" checked={includeVolumeData}>
+                  Volume Annotations as WKW
+                </Checkbox>
+                <Hint
+                  style={{
+                    marginLeft: 24,
+                    marginBottom: 12,
+                  }}
+                >
+                  Download a zip folder containing WKW files.
+                </Hint>
+
+                <Checkbox style={checkboxStyle} value="Skeleton" checked disabled>
+                  Skeleton Annotations as NML
```

Comment:
In case of volume-only annotations this should say `Meta data as NML` (check for `state.tracing.skeleton == null`).